posts/2022-06-24-rust-unconference.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Announcing The Rustconf Day After Unconference"
```

Comment:
Needs a better title

Reply:
RustConf, capital C <3

And if we _do_ stick with this, we should probably hyphenate day-after.